js/zonda.js

```diff
@@ -705,7 +1119,7 @@ module.exports = class zonda extends Exchange {
             request['limit'] = limit;
         }
         request = this.extend (request, params);
-        const response = await this.v1_01PrivateGetBalancesBITBAYHistory ({ 'query': this.json (request) });
+        const response = await this.privateGetBalancesBITBAYHistory ({ 'query': this.json (request) });
         const items = response['items'];
```

Comment:
Humm, the exchange is reporting weird 0 values don't you think?
![image](https://user-images.githubusercontent.com/43336371/184905014-586409d1-bc1f-46b7-9d74-9cfabdb5cf77.png)

Reply:
I don't remind if you were in that thread, where we talked about it, but ZONDA can have multiple "wallets" for same coin (original response is a flat array of objects). Let's say, the 14th item in array is BTC wallet (where you have zero amount), and also 39th item is also of BTC wallet (where you have 123 amount). one of them (probably "primary/default" wallet) overwrites the other. It might end up with such unified response where balance is zero.  so, I don't have any clearcut solution how that should be solved with our 'balance' structure , it needs @lead's decision.  for this PR (not to pedalize/delay it) we might not be addressing this issue, as this current PR is more of overal api update.